app/controllers/concerns/bbb_server.rb

```diff
@@ -54,6 +54,9 @@ def join_path(room, name, options = {}, uid = nil)
     join_opts = {}
     join_opts[:userID] = uid if uid
     join_opts[:join_via_html5] = true
+    if current_user
+      join_opts[:avatarURL] = current_user.image
```

Comment:
This should be `join_opts[:avatarURL] = current_user.image if current_user.image.present?`, that way, we only set the option if the user has an image set

Reply:
Yes, it's a good idea to make sure that nothing unexpected will happen in the future.

Reply:
Didn't pass the rubocop check, complaining that current_user undefined..
Should I do something like:
`join_opts[:avatarURL] = current_user.image if (defined? current_user) && current_user.image.present?`

Reply:
I reverted it to 'join_opts[:avatarURL] = current_user.image if current_user && current_user.image.present?'
This should be fine even if Rubocop complains.